build/localnet/manifests/configs.yaml

```diff
@@ -1699,7 +1699,7 @@ data:
           "address": "001022b138896c4c5466ac86b24a9bbe249905c2",
           "public_key": "56915c1270bc8d9280a633e0be51647f62388a851318381614877ef2ed84a495",
           "chains": ["0001"],
-          "service_url": "servicer-001-pocket:42069",
+          "service_url": "http://servicer-001-pocket:50832",
```

Comment:
why did we update the port?

Reply:
Port `50832` is the [configured RPC port](https://github.com/pokt-network/pocket/blob/40f325305c1756bbfd069bf139fa67545419981c/charts/pocket/values.yaml#L111-L113), so `42069` seems incorrect (encountered this while running the E2E test introduced in this PR)

Reply:
I think we should update all the RPC ports then to avoid a discrepancy:

![Screenshot 2023-07-05 at 4 07 04 PM](https://github.com/pokt-network/pocket/assets/1892194/40f5ed2d-a6d0-497d-a753-3f28f08a988f)

If you want to do it in a small separate PR (while this one is being reviewed), I support that!

Reply:
> I still think a different small "micro PR" could be simpler & faster

Yes, I will open a separate, small PR for this shortly.